packages/pluggableWidgets/signature-native/e2e/specs/Signature.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { element, by } from "detox";
+import { expectToMatchScreenshot, resetDevice, tapMenuItem } from "../../../../../detox/src/helpers";
+
+describe("Signature", () => {
+    beforeEach(async () => {
+        await tapMenuItem("Signature");
+    });
+
+    afterEach(async () => {
+        await resetDevice();
+    });
+
+    it("should be able to save a signature", async () => {
+        await element(by.id("signature")).swipe("up", "slow", 0.4, 0.5, 0.5);
+        await element(by.id("signature")).swipe("right", "slow", 0.4, 0.5, 0.5);
+        await element(by.id("signature")).swipe("down", "slow", 0.4, 0.5, 0.5);
+        await element(by.id("signature")).swipe("left", "slow", 0.4, 0.5, 0.5);
+        await expectToMatchScreenshot();
```

Comment:
I created new screenshots, hope they are better

Reply:
Screenshots are still failing. I would like to propose a few steps:

- Take a screenshot of the element i.e. signature widget rather than a complete page.
- Change the `failureThresholdType ` to `percent` and `failureThreshold` to around `1` 🤔
OR  
- Change the comparison method to `ssim`. More info at [ssim](https://github.com/americanexpress/jest-image-snapshot#recommendations-when-using-ssim-comparison). I did try out as it looks promising but no great success. I'd try more on it again.

Reply:
Maybe we can try the 'allowSizeMismatch' option. No idea how it works exactly. The documentation is not really clear on that.